multinode_integration_tests/src/main.rs

```diff
@@ -489,6 +592,125 @@ mod tests {
         assert_eq!(data_hunk.data, vec!(1, 2, 3, 4));
     }
 
+    #[test]
+    fn can_read_probe_target_from_str() {
+        probe_target_pairs()
+            .into_iter()
+            .for_each(|(string, expected)| {
+                let result = ProbeTarget::from_str(&string).unwrap();
+
+                assert_eq!(result, expected);
+            })
+    }
+
+    #[test]
+    fn display_implementation_works() {
+        probe_target_pairs()
+            .into_iter()
+            .for_each(|(expected, target)| {
```

Comment:
Similar issue with expected. 

The problem is that this kind of simplicity forces the reader to jump away to see the `probe target pairs` and learn about their data types.

Reply:
Fixed